spec/prelude/main.md

```diff
@@ -22,7 +22,7 @@ pub enum TerminationInfo {
     /// The execution encountered undefined behaviour.
     Ub(String),
     /// The program was executed and the machine stopped without error.
-    MachineStop,
+    MachineStop(String),
```

Comment:
As the comment says, this was meant for stopping *without* an error. But now you are using it for stopping with an error as well. That's also why panics now silently work as "pass" tests, which is not great.

Instead, maybe add a new `TerminationInfo` variant called `MachineAbort(String)`, and then minimize's `main.rs` can print the abort message and exit with a non-zero exit code?